.github/workflows/auto_assign.yaml

```diff
@@ -0,0 +1,17 @@
+name: Auto Assign
```

Comment:
I really dont get why this action is needed !

I think this is one time process to add assignees to current opened PRs and we are done. Anyways henceforth the assignees will be added automatically.

Reply:
We needed this because issues are getting created and assigned to SATQE bot user. This is why because we use SATQE bot user to create the autocherrypick PRs. If the PR is already assigned to Auther automatically, which is a valid case mostly, most people don't do that. As we assigned this automatically we can leverage that to create issues with those assignees.